src/App.js

```diff
@@ -22,12 +22,11 @@ const App = (props) => (
           render={() => (
             <Profile
               profilePage={props.state.profilePage}
-              addPost={props.addPost}
-              updateNewPostText={props.updateNewPostText}
+              dispatch={props.dispatch}
```

Comment:
Is it Redux dispatch?
If so, consider connecting Profile component itself to Redux, instead of passing dispatch to it.

Reply:
No it's not an Redux dispatch